src/header/header.scss

```diff
@@ -341,5 +313,95 @@
         }
       }
     }
+
+    &.iui-disabled {
+      .iui-button {
+        &:last-child:hover {
+          @include themed {
+            background-color: transparent;
+          }
+        }
+      }
+    }
+  }
+}
+
+@mixin iui-header-button-active {
+  // Blue icons
+  svg {
+    @include themed {
+      fill: t(iui-icons-color-primary);
+    }
+  }
+
+  &,
+  &:hover {
+    // Blue background and darker blue bottom line
+    @include themed {
+      box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+      background-color: rgba(
+        t(iui-color-foreground-primary-rgb),
+        t(iui-opacity-6)
+      );
+    }
+
+    .iui-header-button-icon {
+      @include themed {
+        fill: t(iui-icons-color-primary);
+        background-color: rgba(
+          t(iui-color-foreground-primary-rgb),
+          t(iui-opacity-6)
+        );
+      }
+    }
+  }
+
+  // Blue text
+  .iui-label {
+    @include themed {
+      color: t(iui-color-foreground-primary);
+    }
+  }
+
+  // Focus styling - blue outline
+  &:focus {
+    @include themed {
+      box-shadow: t(iui-focus-box-shadow),
+        0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+    }
+  }
+
+  &:focus:not(:focus-visible) {
+    @include themed {
+      box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+    }
+  }
+
+  // Disabled state
+  &.iui-disabled,
+  &:disabled {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-body-rgb), 0.05);
+      box-shadow: 0 $iui-xxs 0 0
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
```

Comment:
These values are very random and may not be "correct". Lets decide the visuals once Jon gets back. I will create a discussion.